src/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const newsAPI = axios.create({
+	baseURL: 'https://nc-news-xd69.onrender.com/api',
+});
+
+const fetchArticles = async () => {
+	const results = await newsAPI.get('/articles');
+	const sortedResults = results.data.articles.sort(
```

Comment:
We shouldn't be sorting with array methods when our endpoint can accept a sort query for data. It's more efficient, and this wouldn't be sustainable if we were retrieving large amounts of data. Please change this to use a query instead